tests/python/TestBoundingBox.py

```diff
@@ -0,0 +1,463 @@
+        self.assertEqual(response[0]["AddBoundingBox"]["status"], 0)
+
+    def findAllBoundingBoxes(self, db):
+        all_queries = []
+
+        results = {}
+        results["list"] = ["name", "_coordinates"]
+
+        bb_params = {}
+        bb_params["results"] = results
+
+        query = {}
+        query["FindBoundingBox"] = bb_params
+
+        all_queries.append(query)
+
+        response, img_array = db.query(all_queries)
+
+        self.assertEqual(response[0]["FindBoundingBox"]["status"], 0)
+        print "\n"
+        for i in range(0, len(response[0]["FindBoundingBox"]["entities"])):
+            print response[0]["FindBoundingBox"]["entities"][i]["name"], response[0]["FindBoundingBox"]["entities"][i]["_coordinates"]
+
+    def addBoundingBoxwithImage(self, db, numBoxes, imgprops=None):
+        all_queries = []
+        imgs_arr = []
+
+        fd = open("../test_images/brain.png", 'rb')
+        imgs_arr.append(fd.read())
+        fd.close()
+
+        img = {}
+        img["properties"] = imgprops
+        img["format"] = "png"
+        img["_ref"] = 12
+
+        query = {}
+        query["AddImage"] = img
+
+        all_queries.append(query)
+
+        basename = imgprops["name"] + "_bb_"
+        for x in range(0, numBoxes):
+            bb_coords = {}
+            bb_coords["x"] = x*10
+            bb_coords["y"] = x*10
+            bb_coords["height"] = 100
+            bb_coords["width"] = 100
+
+            bbprops = {}
+            bbprops["name"] = basename + str(x)
+            bb = {}
+            bb["properties"] = bbprops
+            bb["rectangle"] = bb_coords
+            bb["image"] = 12
+
+            bb_query = {}
+            bb_query["AddBoundingBox"] = bb
+            all_queries.append(bb_query)
+
+        response, res_arr = db.query(all_queries, [imgs_arr])
+
+        self.assertEqual(len(response), numBoxes + 1)
+        self.assertEqual(response[0]["AddImage"]["status"], 0)
+        for i in range(0, numBoxes):
+            self.assertEqual(response[i+1]["AddBoundingBox"]["status"], 0)
+
+
+    def test_addBoundingBox(self):
+        db = vdms.vdms()
+        db.connect(hostname, port)
+
+        all_queries = []
+
+        for i in range(0, number_of_inserts):
+            bb_coords = {}
+            bb_coords["x"] = i
+            bb_coords["y"] = i
+            bb_coords["height"] = 512
+            bb_coords["width"] = 512
+
+            props = {}
+            props["name"] = "my_bb_" + str(i)
+
+            bb = {}
+            bb["properties"] = props
+            bb["rectangle"] = bb_coords
+
+            query = {}
+            query["AddBoundingBox"] = bb
+
+            all_queries.append(query)
+
+        response, img_array = db.query(all_queries)
+
+        self.assertEqual(len(response), number_of_inserts)
+        for i in range(0, number_of_inserts):
+            self.assertEqual(response[i]["AddBoundingBox"]["status"], 0)
+
+    def test_findEntityBoundingBox(self):
```

Comment:
this test is not really doing much. it is just testing entities. We already have this tests for regular entities.